funnel/models/user.py

```diff
@@ -192,6 +197,16 @@ def name(cls):  # NOQA: N805
     def is_active(self):
         return self.status == USER_STATUS.ACTIVE
 
+    @cached_property
+    def verified_contact_count(self):
+        count = 0
+        count += len(self.emails)
+        count += len(self.phones)
+        return count
+
+    def has_verified_contact_info(self):
```

Comment:
This can be a property. Also, can't the test simply be `return bool(self.emails or self.phones)`

Reply:
I'm using the count because it's cached already. so will be faster in templates.